src/pages/tutorial/angular/wrapping-up.mdx

```diff
@@ -43,7 +43,7 @@ This badge demonstrates knowledge about Carbon's Angular components. To earn the
 
 ### FAQ and help
 
-Badge Issuer Contact: kellychurchill@us.ibm.com
+Badge Issuer Contact: matt.rosno@ibm.com
```

Comment:
```suggestion
Badge issuer contact: matt.rosno@ibm.com
```

We could also just write this as a sentence with a `mailto` link for your name.

```
Please don't hesitate to [email us](mailto:matt.rosno@ibm.com) with any additional questions.
```

Also wouldn't be the worst idea to use the Carbon team address to future-proof this.

Reply:
Let's keep my name there until I get added to Carbon's team address.